msteams-platform/toolkit/configure-bot-capability.md

```diff
@@ -0,0 +1,522 @@
+          ignoreFile: .webappignore
+          # The resource id of the cloud resource to be deployed to.
+          # This key will be generated by arm/deploy action automatically.
+          # You can replace it with your existing Azure Resource id
+          # or add it to your environment variable file.
+          resourceId: ${{BOT_AZURE_APP_SERVICE_RESOURCE_ID}}
+   ```
+
+   For more information, see the [sample project](https://github.com/OfficeDev/TeamsFx-Samples/tree/dev/hello-world-bot-with-tab).
+
+1. Run `Teams: Provision` command to apply the bicep to Azure.
+1. Run `Teams: Deploy` command to deploy your app code to Azure.
+1. Open the **Run and Debug Activity Panel** and select **Launch Remote (Edge)** or **Launch Remote (Chrome)**.
+1. Select F5 key to debug and preview your Teams app.
+
+# [Add bot to Teams message extension app](#tab/messageextensionapp)
+
+You can add a message extension to a Teams bot app more easily than adding it to a tab app, as message extensions use the bot support framework.
+
+The following steps help you to add bot to a message extension app:
+
+* [Create bot app using Teams Toolkit](#create-bot-app-using-teams-toolkit).
+* [Update manifest file](#add-bot-in-app-manifest).
+* [Add bot code to project](#add-bot-code-to-project).
+
+## Create bot app using Teams Toolkit
+
+To create a bot app with Teams Toolkit, see [create a new Teams app](create-new-project.md).
+
+## Add bot in app manifest
+
+1. You can configure bot in `appPackage/manifest.json`. If you need to customize your app, see the [app manifest schema](../resources/schema/manifest-schema.md#bots).
+
+   ```json
+    "bots": [
+        {
+            "botId": "${{BOT_ID}}",
+            "scopes": [
+                "personal",
+                "team",
+                "groupchat"
+            ],
+            "supportsFiles": false,
+            "isNotificationOnly": false,
+            "commandLists": [
+                {
+                    "scopes": [
+                        "personal",
+                        "team",
+                        "groupchat"
+                    ],
+                    "commands": [
+                        {
+                            "title": "welcome",
+                            "description": "Resend welcome card of this Bot"
+                        },
+                        {
+                            "title": "learn",
+                            "description": "Learn about Adaptive Card and Bot Command"
+                        }
+                    ]
+                }
+            ]
+        }
+    ]
+   ```
+
+## Add bot code to project
+
+If you're adding bot to a message extension Teams app, you should have a class that extends `TeamsActivityHandler`. Add your bot code or copy the code from your previously created bot app to your own class. The following code is an example if you copy the code from a bot app created with Teams Toolkit:
+
+   ```json
```

Comment:
This JSON file isn't color coded. Generally, it is color coded for JSON scripts in review site.